signal-element.js

```diff
@@ -56,36 +56,39 @@ class SignalElement extends HTMLElement {
 			? coerce(this.getAttribute('state')) || this.innerHTML
 			: coerce(this.getAttribute('state')) || coerce(this.textContent);
 		this.signal = new Signal.State(initial);
-		this.cleanup = effect(() => this.render());
+		this.cleanup = effect(() => this._render());
 	}
 	connectedCallback() {
-		this.render();
+		this._render();
 	}
 	disconnectedCallback() {
 		this.cleanup();
 	}
-	render() {
+	_render() {
 		const value = this.mutation(this.signal.get());
 		if (this.isHTML) {
 			this.innerHTML = value;
 		} else {
 			this.textContent = `${value}`;
 		}
 	}
+	render() {
+		return this._render();
+	}
```

Comment:
Curious what your thinking is for including `render` method that returns `this._render()`?

`this._render()` doesn't return anything and will always be run by the `effect` code anyway.

Reply:
Ah, yeah you are right. I think I left this in from some other prototype code I was messing around with.